packages/flutter_tools/lib/src/vmservice.dart

```diff
@@ -539,60 +540,75 @@ class FlutterVmService {
   }
 
   Future<String> flutterDebugDumpApp({
-    @required String isolateId,
+    required String isolateId,
   }) async {
-    final Map<String, Object> response = await invokeFlutterExtensionRpcRaw(
+    final Map<String, dynamic>? response = await invokeFlutterExtensionRpcRaw(
       'ext.flutter.debugDumpApp',
       isolateId: isolateId,
     );
-    return response != null ? response['data']?.toString() : '';
+    if (response != null) {
+      return response['data']?.toString() ?? '';
+    }
+    return '';
   }
 
   Future<String> flutterDebugDumpRenderTree({
-    @required String isolateId,
+    required String isolateId,
   }) async {
-    final Map<String, Object> response = await invokeFlutterExtensionRpcRaw(
+    final Map<String, dynamic>? response = await invokeFlutterExtensionRpcRaw(
       'ext.flutter.debugDumpRenderTree',
       isolateId: isolateId,
       args: <String, Object>{}
     );
-    return response != null ? response['data']?.toString() : '';
+    if (response != null) {
```

Comment:
etc, same comment about the `flutterDebugDumpApp` return statement.

Reply:
done